src/silhouette.jl

```diff
@@ -70,13 +99,48 @@ function silhouettes(assignments::AbstractVector{<:Integer},
             end
         end
     end
+    return r
+end
+
+function sil_aggregate_dists_normalized(assignments::AbstractVector{<:Integer},
```

Comment:
I'm not sure we need these function in addition to `sil_aggregate_dist`, the only thing it does is calling `normalize...()`

Reply:
The purpose here was to test that the point to cluster distances were consistent with the baseline code, before the normalization.
Since this is not something the user needs to worry about, I think we should leave this, as this provides another important layer of testing.